cypress/e2e/Labels/utilsAndConstants/labelsFormUtils.ts

```diff
@@ -14,3 +14,70 @@ export const checkPresenceOfGroupNameInput = (options: {
       }
     });
 };
+
+export const fillInNewLabelForm = (options: {
+  groupName?: string;
+  labelName?: string;
+  condition: string;
+  rank?: string;
+}) => {
+  const { groupName, labelName, condition, rank } = options;
+
+  labelName && cy.get('input[aria-label="Label Name"]').clear().type(labelName);
+
+  groupName &&
+    groupName !== UNGROUPED &&
+    cy.get('input[aria-label="Group Name"]').clear().type(groupName);
+
+  rank &&
+    cy.get('input[aria-label="Group Rank"]').clear().type(rank).type("{enter}");
+
+  cy.get('textarea[aria-label="Condition"]')
+    .clear()
+    .type(condition)
+    .type("{command+enter}");
```

Comment:
will this work on non-mac?

Reply:
@doronrosenberg , if you are referring to `command+enter` then yes, it works on mac and in GHA.
This was the fix for https://github.com/sundeck-io/OpsCenter/issues/39 (all the details on implementation are there)